.github/workflows/create-docs-pr.yml

```diff
@@ -5,20 +5,27 @@
 #   `documentation-gitbook` is behind           Open A PR from `documentation-gitbook` into `main`   Determine the directionality, if wrong dismiss and open a PR from `documentation-gitbook` into `main`   Dismiss the PR from `main` into `documentation-gitbook`  
 #   `documentation-gitbook` is ahead & behind   Open 2 PRs                                           Determine the directionality of the missing one and open that                                           Nothing                                                  
 
-name: create-docs-pr
+name: Create GitBook Sync PRs
 
 on:
   workflow_dispatch:
-  schedule:
-    - cron: '1 10 * * 1-5'
+  pull_request:
+    types:
+      - closed
+  push:
+    branches:
+      - documentation-gitbook
+    paths:
+      - documentation/**
 
 jobs:
-  create-pr:
+  create-prs:
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || (github.event.pull_request.merged == true && (github.event.pull_request.base.ref == 'main' || github.event.pull_request.base.ref == 'documentation-gitbook'))
     env:
       DOCS_BRANCH: documentation-gitbook
       PR_LABELS: 'documentation,high priority'
-      PR_ASSIGNEES: 'emprzy,jcblemai,pearsonca,saraloo,TimothyWillard'
+      PR_REVIEWERS: 'emprzy,jcblemai,pearsonca,saraloo,TimothyWillard'
```

Comment:
@saraloo is good here, but this needs to be more of the operations folks and less the devs - @alsnhll, @twallema, @MacdonaldJoshuaCaleb, @shauntruelove probably? not me, you, Emily

Reply:
Adding the ability to get members from a team to dynamically assign them to review would require using a custom token which I do not have the permission to setup: https://github.com/actions/github-script/discussions/481.